package.json

```diff
@@ -7,19 +7,19 @@
   "description": "Dat Desktop App",
   "author": "Dat Team <community@datproject.org>",
   "scripts": {
-    "start": "cross-env NODE_V=\"$(node -v)\" NODE_ENV=development electron .",
-    "pretest": "npm run lint",
-    "test": "npm run test:unit",
+    "start": "cross-env NODE_ENV=development npm-run-all -p watch start:dev",
+    "start:dev": "cross-env NODE_V=\"$(node -v)\" NODE_ENV=development electron .",
```

Comment:
Maybe name "start: electron"?

Reply:
Yes, I had the same idea first. Will change.